src/applications/static-pages/health-care-manage-benefits/components/CernerCallToAction/index.js

```diff
@@ -84,7 +85,18 @@ export class CernerCallToAction extends Component {
     }
   };
 
+  onCTALinkClick = (buttonLabel, buttonType = 'default') => () => {
+    recordEvent({
+      event: 'cta-button-click',
+      'button-type': buttonType,
+      'button-click-label': buttonLabel,
+      'button-background-color':
+        buttonType === 'secondary' ? '#ffffff' : '#0071bb',
+    });
+  };
```

Comment:
I saw this button-bg-color thing another time during review and this little `getComputedStyle` did the trick around those hardcoded colors.

```
const style = window.getComputedStyle(event.target);
      recordEvent({
        event: 'cta-button-click',
        'button-type': getButtonType(event.target.classList),
        'button-click-label': event.target.text,
        'button-background-color': style.getPropertyValue('background-color'),
      });
```

As a Platform task though, this should totally be abstracted though into a shared button component or something, esp since I think the intent is to do this for every button on the website.

Reply:
Huh, that's pretty cool! I'll use that, thanks! 🙂

Reply:
Updated